staking/types/validator.go

```diff
@@ -77,26 +78,64 @@ type counters struct {
 	NumBlocksSigned *big.Int `json:"num-blocks-signed",rlp:"nil"`
 }
 
-// ValidatorWrapper contains validator and its delegation information
+// ValidatorWrapper contains validator,
+// its delegation information
 type ValidatorWrapper struct {
 	Validator
 	Delegations Delegations
-	Counters    counters
+	//
+	Counters counters
+	// All the rewarded accumulated so far
+	BlockReward *big.Int
 }
 
 // Computed represents current epoch
 // availability measures, mostly for RPC
 type Computed struct {
-	Signed     *big.Int    `json:"current-epoch-signed"`
-	ToSign     *big.Int    `json:"current-epoch-to-sign"`
-	Percentage numeric.Dec `json:"percentage"`
+	Signed            *big.Int    `json:"current-epoch-signed"`
+	ToSign            *big.Int    `json:"current-epoch-to-sign"`
+	BlocksLeftInEpoch uint64      `json:"current-epoch-blocks-left"`
```

Comment:
BlocksLeftInEpoch remove?

Reply:
Its useful, people downstream asking for it and good to know how much blocks left for me to go